doc/Learn/Tutorials/Navigation/Advanced/HowTo-UsePanel.md

```diff
@@ -1,23 +1,252 @@
+  </Grid>
+  ```
+
+* Your `Page` element should now look like this:
+
+  ```xml
+  <Page x:Class="UsingPanelRegion.Views.MainPage"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:local="using:UsingPanelRegion.Views"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        mc:Ignorable="d"
+        xmlns:uen="using:Uno.Extensions.Navigation.UI"
+        xmlns:utu="using:Uno.Toolkit.UI"
+        Background="{ThemeResource MaterialBackgroundBrush}">
 
-#### Navigating using a Panel
+      <Grid>
+          <Grid.RowDefinitions>
+              <RowDefinition Height="Auto" />
+              <RowDefinition Height="Auto" />
+              <RowDefinition />
+          </Grid.RowDefinitions>
+
+          <utu:NavigationBar Content="Main Page" 
+                            Style="{StaticResource MaterialNavigationBarStyle}"/>
+
+          <StackPanel Grid.Row="1" 
+                      HorizontalAlignment="Center" 
+                      Orientation="Horizontal">
+              <Button Content="One" />
+              <Button Content="Two" />
+              <Button Content="Three" />
+          </StackPanel>
+
+          <Grid Grid.Row="2">
+              <Grid>
+                  <TextBlock Text="One" 
+                            FontSize="24" 
+                            HorizontalAlignment="Center" 
+                            VerticalAlignment="Center"/>
+              </Grid>
+              <Grid>
+                  <TextBlock Text="Two"
+                            FontSize="24"
+                            HorizontalAlignment="Center"
+                            VerticalAlignment="Center" />
+              </Grid>
+              <Grid>
+                  <TextBlock Text="Three"
+                            FontSize="24"
+                            HorizontalAlignment="Center"
+                            VerticalAlignment="Center" />
+              </Grid>
+          </Grid>
+      </Grid>
+  </Page>
+  ```
+
+### 3. Set up navigation
+
+* Set the `Navigation.Request` attached property on each `Button` control to what will be the route to a corresponding view:
+
+  ```xml
+  <StackPanel Grid.Row="1" 
+              HorizontalAlignment="Center" 
+              Orientation="Horizontal">
+      <Button Content="One"
+              uen:Navigation.Request="./One" />
+      <Button Content="Two"
+              uen:Navigation.Request="./Two" />
+      <Button Content="Three"
+              uen:Navigation.Request="./Three" />
+  </StackPanel>
+  ```
+
+* This will automatically set up navigation to the corresponding view when the `Button.Click` event is raised.
+
+* Next, we need to attach content regions for each view. The first step is to set the `Region.Attached` attached property to `True` on the parent `Grid` containing the views:
```

Comment:
Setup the navigation region before defining the Navigation.Requests. Need to explain that this region is a container for where navigation will take place.